internal/config/0_common_config.go

```diff
@@ -12,9 +12,9 @@ func init() {
 	// Database
 	parser.Viper.SetDefault("database.host", "localhost")
 	parser.Viper.SetDefault("database.port", 5432)
-	parser.Viper.SetDefault("database.name", "provisioning")
```

Comment:
I thought we are using this default in GH accion, but apparently not, so lets get rid of it ^_^